tests/test_google_analytics_pagination.py

```diff
@@ -2,93 +2,125 @@
 ### TODO WAITING ON https://stitchdata.atlassian.net/browse/SRCE-5065
 ##########################################################################
 
-# import os
 
-# from datetime import timedelta
-# from datetime import datetime as dt
+from math import ceil
+from datetime import timedelta
+from datetime import datetime as dt
 
-# from  tap_tester import connections, runner
+from  tap_tester import connections, runner
 
-# from base import GoogleAnalyticsBaseTest
+from base import GoogleAnalyticsBaseTest
 
 
-# class GoogleAnalyticsPaginationTest(GoogleAnalyticsBaseTest):
-#     """Test that we are paginating for streams when exceeding the API record limit of a single query"""
+class GoogleAnalyticsPaginationTest(GoogleAnalyticsBaseTest):
+    """Test that we are paginating for streams when exceeding the API record limit of a single query"""
 
-#     @staticmethod
-#     def name():
-#         return "tap_tester_google_analytics_pagination_test"
+    # TODO https://stitchdata.atlassian.net/browse/SRCE-5084
```

Comment:
This seems to be an old link (old JIRA environment).
Is there a new JIRA associated with it, if not can we remove this ?

Reply:
@cosimon @kspeer825 Should we remove such a Jira link from the tap-tester? We are not sure whether it is an old Jira link or not because it is not accessible to us.